std/meta.d

```diff
@@ -1024,3 +1024,55 @@ unittest
 // TODO: Consider publicly exposing this, maybe even if only for better
 // understandability of error messages.
 alias Instantiate(alias Template, Params...) = Template!Params;
+
+private template staticMerge(alias C, int half, T...)
+{
+    static if (half == 0 || half == T.length)
+    {
+        alias staticMerge = T;
+    }
+    else
+    {
+        static if (C!(T[0], T[half]))
+        {
+            alias staticMerge = AliasSeq!(T[0], staticMerge!(C, half - 1, T[1 .. $]));
+        }
+        else
+        {
+            alias staticMerge = AliasSeq!(T[half], staticMerge!(C, half,
+                T[0 .. half], T[half + 1 .. $]));
+        }
+    }
+}
+
+/**
+ * Sorts a tuple using $(D C).
+ *
+ * Where $(D C) is a template that returns if it's first argument is less than it's second
+ */
+template staticSort(alias C, T...)
```

Comment:
So that technically allows sorting a tuple of types - add an exampe for that? Otherwise for plain integers a function runable at CTFE should be enough.